tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -939,12 +939,12 @@ def compile_wasm_to_aot(wasm_tempfile, aot_tempfile, runner, opts, r):
     if opts.sgx:
         cmd.append("-sgx")
 
-    if opts.simd:
-        cmd.append("--enable-simd")
+    if not opts.simd:
+        cmd.append("--disable-simd")
 
-    if opts.ref_types:
-        cmd.append("--enable-ref-types")
-        cmd.append("--enable-bulk-memory")
+    if not opts.ref_types:
+        cmd.append("--disable-ref-types")
+        cmd.append("--disable-bulk-memory")
```

Comment:
Should not disable bulk memory here, in other feature e.g. thread, it must be enabled.